api/v1alpha2/EtcdManagerSpec.dhall

```diff
@@ -1 +1,4 @@
-{ Type = { image : Optional Text }, default.image = None Text }
+{ Type =
+    { image : Optional Text, env : Optional { mapKey : Text, mapValue : Text } }
+, default = { image = None Text, env = None { mapKey : Text, mapValue : Text } }
```

Comment:
wrong type for `env`, it should be `Optional (List { name : Text , value : Text })`